### PR TITLE
Updated the Registry Key Path for Windows Settings

### PR DESCRIPTION
**Pull request by AdamGrossTX**
Changed the entry from 
Microsoft\Windows\DataCollection
to
HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\DataCollection

### Summarize the change in the pull request title

Describe your change, specifically *why* you think it's needed.

Fixes #Issue_Number (if necessary)

### CODE REVIEW

**Comment by PRMerger14** [comment]
@AdamGrossTX : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @AdamGrossTX as always 😄 
But I do want to double-check with @iceorbital - I want to say we *purposefully* left it like that because it can appear in two different places depending upon how it's set (and at the time we didn't want to get into the gory details - we want folks to use ConfigMgr for these settings, not other mechanism). But I could also be misremembering - stay tuned.

File: sccm/desktop-analytics/enroll-devices.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ To change these settings, use the following procedure:
 
 ### Windows settings
 
-Configuration Manager sets the following Windows settings under `Microsoft\Windows\DataCollection`:
+Configuration Manager sets the following Windows settings under `HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\DataCollection`:
```
Comment (aczechowski) [inline sccm/desktop-analytics/enroll-devices.md]: SCCM actually sets the local policy path, not the global:

```suggestion
Configuration Manager sets the following Windows settings under the local policy path `HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\DataCollection`:
```

**Comment by PRMerger15** [comment]
@AdamGrossTX : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by AdamGrossTX** [comment]
Thanks for the follow up. I had actually gotten the same clarification from Premiere Support and was going to make the same edits. Thanks for the quick response!

**Comment by aczechowski** [comment]
#sign-off 

*Once merged*, this should be live with the next daily publishing process. Right now that *should* be today (Friday, 26 July) around 11:00 Pacific time. No further action needed from you.

**Pull request merged**